pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+    - 'example'
```

Comment:
now or later: I think we'll want to pluralize "examples" and have a package.json under each, like the mocha one in https://github.com/aspect-build/rules_js/pull/119

Reply:
sounds like a follow-up